ui/app/components/transaction-list-item.js

```diff
@@ -44,27 +44,43 @@ TransactionListItem.prototype.render = function () {
 
   return (
     h(`.transaction-list-item.flex-row.flex-space-between${isClickable ? '.pointer' : ''}`, {
-      onClick: (event) => {
```

Comment:
Why remove the global listener?  Doesn't this mean only very specific clicks will go to the explorer now?

I figure this is probably because you were finding the click would trigger both links.

The way to avoid this is to use the `event.stopPropagation()` method in the top level listener, so it prevents the click event from propagating up the DOM tree.